### PR TITLE
Include result link in job slack notification

### LINKED ISSUE #5857: In Job Slack notification, include link to result
It would be helpful to debug bad job results quicker if the job notifications sent to slack could already include the result link. This link is currently constructed by the frontend on the jobs list view, it could instead be constructed by the back-end (and then be sent to slack and to the frontend).

## Context
* [x] Specific to long-running jobs (set `jobsEnabled=true` in `application.conf`)
* [x] Specific to webKnossos.org (set `isDemoInstance=true` in `application.conf`)

### PR DESCRIPTION
**Pull request by fm3**
The result link was hitherto constructed in the front-end. Now I construct it in the back-end instead and send it both to the front-end and to the slack notification.

### Steps to test:
- set up jobs locally
- run jobs of different type
- result links in jobs page should still work
- job slack notifications should include link (set slackNotifications.verboseLoggingEnabled = true to test)

### Issues:
- fixes #5857 

------
- [x] Ready for review

### CODE REVIEW

**Comment by fm3** [comment]
@philippotto `activeUser` is now never used in the jobs_list_view. I did not manage to remove it without angering flow. Maybe you could have a quick look :see_no_evil: no hurry :)

**Comment by philippotto** [comment]
> @philippotto activeUser is now never used in the jobs_list_view. I did not manage to remove it without angering flow. Maybe you could have a quick look see_no_evil no hurry :)

Good catch! It's removed now.

File: app/models/job/Job.scala [hunk 1]
```diff
@@ -223,14 +254,16 @@ class JobService @Inject()(wkConf: WkConf,
     ()
   }
 
-  private def trackNewlySuccessful(jobBeforeChange: Job, newStatus: JobStatus): Unit = {
+  private def trackNewlySuccessful(jobBeforeChange: Job, jobAfterChange: Job): Unit = {
     for {
       user <- userDAO.findOne(jobBeforeChange._owner)(GlobalAccessContext)
       organization <- organizationDAO.findOne(user._organization)(GlobalAccessContext)
+      resultLink = jobAfterChange.resultLink(organization.name)
+      resultLinkMrkdwn = resultLink.map(l => s" <${wkConf.Http.uri}$l|Result Link>").getOrElse("")
```
Comment (philippotto) [inline app/models/job/Job.scala:262]:
```suggestion
      resultLinkMrkdwn = resultLink.map(l => s" <${wkConf.Http.uri}$l|Result>").getOrElse("")
```

If that's the clickable link, I think "Link" can be omitted.

**Review by philippotto — APPROVED** [review]
Great! Looking really forward to use this :100:  Didn't test, since I trust your testing.

**Pull request merged**